substrait/substrait-spark/src/main/resources/spark.yml

```diff
@@ -0,0 +1,20 @@
+%YAML 1.2
+---
+scalar_functions:
+  -
+    name: year
+    description: Returns the year component of the date/timestamp
+    impls:
+      - args:
+          - value: date
+        return: i32
+  -
+    name: unscaled
+    description:  >-
+      Return the unscaled Long value of a Decimal, assuming it fits in a Long.
+      Note: this expression is internal and created only by the optimizer, 
+      we don't need to do type check for it.
+    impls:
+      - args:
+          - value: DECIMAL<P,S>
+        return: i64
```

Comment:
Does current logic rely on this yml file? Looks like TPC-H related functions are not fully added here.

Reply:
Those functions are included substrait-core